packages/frontend/src/components/FilterBar/FilterButton/FilterButton.tsx

```diff
@@ -228,11 +229,7 @@ export const FilterButton = ({
                   </div>
                 }
                 rightContent={
-                  option.options?.length ? (
-                    <ChevronRightIcon fontSize="1.5rem" />
-                  ) : (
-                    <span className={styles.filterListCount}>{option.count}</span>
-                  )
+                  option.options?.length ? <ChevronRightIcon fontSize="1.5rem" /> : <Badge label={option.count} />
```

Comment:
Litt på tynn is ettersom det er en stund siden jeg har jobbet med det, men kan ikke conditionals i komponenter trigge unødvendige rerenders? Uten for mye styrke i argumentet her, så *føler* jeg det er ryddigere å plassere conditionals ut av selve komponent-templatet 🤔 

rightContent={renderRightContent(option.options?.length)} eller assign til en variabel.. 🤔

Nei, jeg skal sette meg litt mer inn i det før jeg påstår mer

Reply:
Det kan, men tror ikke det gjør det her. Ofte vil det overkomplisere å memoisere alt. Har ikke kjørt analyse på det, og dette er jo nå som bare blir bedre og bedre i React. Ikke noe vi trenger å tenke på React 19 :)